src/app/models/project.ts

```diff
@@ -54,6 +54,8 @@ export class Project {
 
   // Options
   activitiesAndUpdatesEnabled: Boolean;
+  contactFormEnabled: Boolean;
+  contactFormEmails: string[];
 
   // Permissions.
   read: Array<String> = [];
```

Comment:
Its recommended to use lowercase string types in TypeScript string[] same like the above line contactFormEmails: string[] instead of Array<String>

Reply:
You're right! I'll update